src/operator/combineLatest.ts

```diff
@@ -48,8 +69,25 @@ export function combineLatest<T, R>(...observables: Array<Observable<any> |
  * @returns {Observable} an observable of other projected values from the most recent values from each observable, or an array of each of
  * the most recent values from each observable.
  */
-export function combineLatestStatic<T, R>(...observables: Array<any | Observable<any> |
-                                                    Array<Observable<any>> |
+/* tslint:disable:max-line-length */
+export function combineLatestStatic<T>(v1: ObservableInput<T>): Observable<[T]>;
```

Comment:
Since these have been merged, I added definitions for `combineLatestStatic` that mirror the signatures.